internal/pkg/levee/levee.go

```diff
@@ -94,20 +118,95 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
+func doPointerAnalysis(pass *analysis.Pass, analysisConf *config.Config, c *ssa.Call) (analyzedSuccesfully bool) {
+	// Pointer analysis requires at least one main package containing a main function.
+	// If we fail to provide such a package, the analysis will fail and produce error output in stderr.
+	pkg := c.Parent().Pkg
+	main := pkg.Func("main")
+	if main == nil {
+		return false
+	}
+
+	// This configuration is used to tell the pointer package how we want the analysis to be performed.
+	// For our purposes, we just need to give it a main package and a set of values for which we want
+	// the points-to set to be computed.
+	pointerConf := &pointer.Config{
+		Mains: []*ssa.Package{pkg},
+	}
+
+	// In general, we can't use the call's arguments as values directly. For example, when calling a variadic
+	// function, the variadic argument will be a slice, but we are not interested in the slice's points-to set,
+	// rather we are interested in the points-to sets of the values in the slice.
+	vs := callValues(c)
+	if len(vs) == 0 {
+		return false
+	}
+
+	for _, v := range vs {
+		// If we ask the pointer package to analyze a value it can't analyze (i.e., not a pointer or
+		// pointer-like type), the analysis will fail and produce error output in stderr.
+		if !pointer.CanPoint(v.Type()) {
+			continue
+		}
+		// Let the configuration know that we are interested in computing this value's points-to set.
```

Comment:
Is `points-to set` is a set of all call-sites which may contribute to the `v` under analysis?

Reply:
It is the set of values that may be pointed to by the given pointer value. I've added that to the comment.